modules/c++/re/include/re/PCRE.h

```diff
@@ -171,10 +187,22 @@ class PCRE
 
 protected:
```

Comment:
I know this predates your changes but any reason this can't be private rather than protected?